common_patches/i2c0.yaml

```diff
@@ -0,0 +1,28 @@
+COMD%s:
+    _delete: ["COMMAND"]
+    _add:
+      OPCODE:
+        description: "Opcode part of command %s. 0: RSTART, 1: WRITE, 2: READ, 3: STOP, 4: END."
```

Comment:
Not a huge deal, feel free to ignore, but probably it would be nice to add an enum here for the different opcodes?